src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -504,9 +505,10 @@ def cli_networkrule_delete(client, resource_group_name, namespace_name, subnet=N
         virtualnetworkrule.subnet = subnet
 
         for vnetruletodelete in netwrokruleset.virtual_network_rules:
-            if vnetruletodelete.subnet.id == subnet:
+            if vnetruletodelete.subnet.id.lower() == subnet.lower():
```

Comment:
Just to double check, someone can remove a vnet rule by providing `--subnet NAME --vnet-name NAME` in addition to `--subnet ID`, right? I know that's already the case for `add`.